apps/prairielearn/src/pages/instructorCourseAdminInstances/instructorCourseAdminInstances.ts

```diff
@@ -43,7 +43,7 @@ router.get(
       const row = enrollmentCounts.find((row) => idsEqual(row.course_instance_id, ci.id));
       ci.number = row?.number || 0;
     });
-    res.render(__filename.replace(/\.js$/, '.ejs'), res.locals);
+    res.send(InstructorCourseAdminInstances({ resLocals: res.locals }));
```

Comment:
There are some things we're attaching to `res.locals` above that could instead be passd explicitly to `InstructorCourseAdminInstances` via separate properties.

Reply:
Moved `courseInstances` from `res.locals` to a local variable in [86e4395](https://github.com/PrairieLearn/PrairieLearn/pull/9812/commits/86e4395ef9e0858a540dadab76ffab2835b365d8).